microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -176,89 +176,31 @@ For more information, see [Welcome to the Mixed Reality Feature Tool](/windows/m
-
-1. In the **Inspector** window for WebView, click **Add Component**, select **New script**, type **WebViewBrowser**, and then click **Create and add**. Your new component is added to the **Inspector** window.
-
-1. In the **Inspector** window, double-click the `WebViewBrowser` script, to edit the script.
-
-1. Replace the contents of that file with the following code:
-
-   ```C#
-   using Microsoft.MixedReality.WebView;
-   using UnityEngine.UI;
-   using UnityEngine;
-   using TMPro;
-   using System;
-
-   public class WebViewBrowser: MonoBehaviour
-   {
-      // Declare UI elements: Back button, Go button, and URL input field
-      public Button BackButton;
-      public Button GoButton;
-      public TMP_InputField URLField;
+## Step 6 - Test the project
 
-      private void Start()
-      {
-         // Get the WebView component attached to the game object
-         var webViewComponent = gameObject.GetComponent<WebView>();
-         webViewComponent.GetWebViewWhenReady((IWebView webView) =>
-         {
-               // If the WebView supports browser history, enable the Back button
-               if (webView is IWithBrowserHistory history)
-               {
-                  // Add an event listener for the Back button to navigate back in history
-                  BackButton.onClick.AddListener(() => history.GoBack());
-
-                  // Update the Back button's enabled state based on whether there's any history to go back to
-                  history.CanGoBackUpdated += CanGoBack;
-               }
-
-               // Add an event listener for the Go button to load the URL that was entered in the input field
-               GoButton.onClick.AddListener(() => webView.Load(new Uri(URLField.text)));
+Now that you have added the WebView plugin to the scene, it would be a good time to test your project.   You can test the project directly in the editor, or you can follow these steps to [Build and deploy to the HoloLens](/windows/mixed-reality/develop/unity/build-and-deploy-to-hololens)
 
-               // Subscribe to the Navigated event to update the URL input field whenever a navigation occurs
-               webView.Navigated += OnNavigated;
+<!-- ====================================================================== -->
+## Step 7 - Extending WebView2 functionality
 
-               // Set the initial value of the URL input field to the current URL of the WebView
-               if (webView.Page != null)
-               {
-                  URLField.text = webView.Page.AbsoluteUri;
-               }
-         });
-      }
+While the prefab exposes just a single property, there is more functionality exposed on the `WebView` script. Let's take a look at exposing some of this functionality in our project.  First, we recommend reviewing the `WebView` script to get an idea of what's available.  
```

Comment:
```suggestion
Although the prefab exposes just a single property, there is more functionality exposed on the `WebView` script. Let's take a look at exposing some of this functionality in our project.  First, we recommend reviewing the `WebView` script to get an idea of what's available.  
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/while - "Don't use [_while_] as a synonym for _although_"